stock_predictor/service.py

```diff
@@ -46,7 +46,12 @@ def load_stock_list():
         query = Query()
         database.upsert(stock_json, query.id == stock.id)
 
-def predict_all(date=None, force_update=False):
```

Comment:
this change is because in a batch i cannot understand if a single row does not have a prediciton then i overwrite the entire batch because of it.

Reply:
@jingedawang please at your leisure, just comment where you want me to revert the code and ill try! i can work on it IST night so no worries just tell me will complete ASAP